sky/skylet/constants.py

```diff
@@ -44,7 +44,7 @@
 # e.g., when we add new events to skylet, or we fix a bug in skylet.
 #
 # TODO(zongheng,zhanghao): make the upgrading of skylet automatic?
-SKYLET_VERSION = '4'
+SKYLET_VERSION = '5'
```

Comment:
It seems we don't need to increase the version number as the new service related event is not needed for the existing clusters?